File: book-network/src/main/java/dev/aniketkadam/book/feedback/FeedbackResponse.java
package dev.aniketkadam.book.feedback;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class FeedbackResponse {
    private double note;
    private String comment;
    private boolean ownFeedback;
    private String fullName;
}

File: book-network-ui/src/models/FeedbackResponse.js
class FeedbackResponse {
    #username;
    #note;
    #comment;
    #ownFeedback;

    constructor(username, note, comment, ownFeedback) {
        this.#username = username;
        this.#note = note;
        this.#comment = comment;
        this.#ownFeedback = ownFeedback;
    }

    get username() {
        return this.#username;
    }

    get note() {
        return this.#note;
    }

    get comment() {
        return this.#comment;
    }

    get ownFeedback() {
        return this.#ownFeedback;
    }
}

export default FeedbackResponse;